sdk/src/error.rs

```diff
@@ -291,6 +294,9 @@ pub enum Error {
     #[error("prerelease content detected")]
     PrereleaseError,
 
+    #[error("capability is not supported by this version")]
+    VersionCompatibility(String),
```

Comment:
If it's `VersionCompatibility(String)`, don't we want the string params also displayed into the error text?

Reply:
added the string